.buildkite/scripts/steps/checks.sh

```diff
@@ -14,4 +14,3 @@ fi
 .buildkite/scripts/steps/code_generation/security_solution_codegen.sh
 .buildkite/scripts/steps/openapi_bundling/security_solution_openapi_bundling.sh
```

Comment:
nit: It looks like `.buildkite/scripts/steps/openapi_bundling/security_solution_openapi_bundling.sh` script could be moved to `.buildkite/scripts/steps/checks/capture_oas_snapshot.sh` since it has a similar purpose but limited to Security Solution only.